src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,24 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+    protected WebDriver driver;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(xpath = "//span[@class='b-main-navigation__text'][text()='Каталог']")
```

Comment:
it's bad idea to find by text, even partially
if `//span[@class='b-main-navigation__text']` finds only one element please use that